stable/artifactory/values.yaml

```diff
@@ -66,6 +66,19 @@ artifactory:
     # version:
     pullPolicy: IfNotPresent
 
+  ## Add custom init containers
+  customInitContainers: |
+  #  - name: "custom-setup"
+  #    image: "{{ .Values.initContainerImage }}"
+  #    imagePullPolicy: "{{ .Values.artifactory.image.pullPolicy }}"
+  #    command:
+  #      - 'sh'
+  #      - '-c'
+  #      - 'touch {{ .Values.artifactory.persistence.mountPath }}/example-custom-setup'
+  #    volumeMounts:
+  #      - mountPath: "{{ .Values.artifactory.persistence.mountPath }}"
+  #        name: artifactory-volume
```

Comment:
Same here
https://github.com/jfrog/charts/blob/09878bed4b355e93b35433ecb0a712db2168c2ee/stable/artifactory-ha/templates/artifactory-primary-statefulset.yaml#L214

Reply:
Here it's right. No need to fix.